packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -276,6 +278,20 @@ export async function publicSettings(
     // settings
     const configDoc = await configs.getSettingsConfigDoc()
     const config = configDoc.config
+
+    // License/Feature Checks
+    const license = await licensing.getLicense()
+
+    const licensedConfig: any = {}
+    if (!license || license?.features.indexOf(Feature.BRANDING) == -1) {
```

Comment:
You can instead use:

```ts
import { features } from "@budibase.pro"
if (!(await features.isFeatureEnabled(Feature.BRANDING)))
```

or you can add a helper to here:
https://github.com/Budibase/budibase-pro/blob/develop/packages/pro/src/sdk/features/features.ts
similar to `isBackupsEnabled`

```ts
export function isBrandingEnabled() {
   return isFeatureEnabled(Feature.BRANDING)
}
```

Reply:
Under the hood this will load the cached license, so will be able to remove the manual license retrieval from above

Reply:
Nice!